src/schema/resolvers/Resolvers.ts

```diff
@@ -78,9 +78,15 @@ export class Resolvers {
         return await UserGroup.findOne({ where: { id } });
     }
 
-    @Query(() => [UserGroup], { nullable: false, description: 'All groups of users' })
+    @Query(() => [UserGroup], { nullable: false, description: 'All groups of users at the ecoverse level' })
```

Comment:
Business logic, should be in a service.

Reply:
Claraifying + fixing the api - current implementation is simply wrong.